.github/workflows/pythonpackage.yml

```diff
@@ -20,15 +20,15 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pipenv
-        pipenv install --dev
+        pip install poetry
+        poetry install
     - name: Lint with flake8
```

Comment:
@mcgeestephen 
Is this workflow intended to publish the python package or as general CI? 

We might want a conditional step at the end that publishes if a build parameter is true (so you can toggle it on manually) or on the main branch to the pypi repo

Reply:
@timothyjlaurent circling back to this (late). It's meant for general CI. There is a `.github/workflows/pythonpublish.yaml` that is meant for this function. Are you thinking we should consolidate the two workflows?